src/s3fs_help.cpp

```diff
@@ -263,6 +263,17 @@ static const char help_string[] =
     "        space is smaller than this value, s3fs do not use diskspace\n"
     "        as possible in exchange for the performance.\n"
     "\n"
+    "   fake_diskfree (default 0)\n"
```

Comment:
Do we need this help string?

Reply:
The man page was confusing and should have been removed.
I'm a little worried about the help string.
The help string just remembers "what is this option for?", So it's unnecessary if this PR remains.(same reason as man page)

Since there is a history of this PR, do you want to delete it?

Reply:
We could leave the help string but enclose it in an `#if 0 .. #endif` pair so it is not included?  I think a more robust solution would be a `--help-hidden` flag but think seems like too much work.

Reply:
Since the PR history remains on github and we can see the code, I will delete this part as well.
(Because putting #ifdef in the code too much is confusing during maintenance)
Thanks.